NeverGreenGrove/NeverGreenGrove/src/DrawableObject.cpp

```diff
@@ -92,5 +92,13 @@ void DrawableObject::Draw(const RenderMode& render_mode) const
       glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, 0);
       glBindVertexArray(0);
       break;
+
+   case RenderMode::TRIANGLE_STRIPS:
+	   glBindVertexArray(m_VAO);
+	   glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, m_Indicies);
+	   glDrawElements(GL_TRIANGLE_STRIP, m_NumVertices * (GLsizei)2 -(GLsizei)382, GL_UNSIGNED_INT, 0);
```

Comment:
ahh maybe if you change this to m_indicies youll have better luck without hardcoding??